hs_core/models.py

```diff
@@ -1757,6 +1757,21 @@ def set_public(self, value, user=None):
                 if value and settings.RUN_HYRAX_UPDATE and self.resource_type == 'NetcdfResource':
                     run_script_to_update_hyrax_input_files(self.short_id)
 
+    def set_download_agreement(self, user, value):
+        """Set resource download_agreement flag to True or False. If download_agreement is True
```

Comment:
@pkdash Perhaps rename this method to ```set_download_agreement_flag```? ```set_download_agreement``` makes one think it is not a flag to set, but set agreement text or something.